packages/amplify-category-auth/src/commands/auth/override.ts

```diff
@@ -18,14 +18,28 @@ export const run = async (context: $TSContext) => {
     printer.error(errMessage);
     return;
   }
-
-  const selectedAuthResource = await prompter.pick<'one', string>(`Which resource would you like to add overrides for?`, authResources);
+  let selectedAuthResource = await prompter.pick<'one', string>(`Which resource would you like to add overrides for?`, authResources);
   // check if migration needed
+  let userPoolGroupResource = undefined;
+  if (selectedAuthResource === 'userPoolGroups') {
+    selectedAuthResource = authResources.filter(resource => resource !== 'userPoolGroups')[0];
+    userPoolGroupResource = 'userPoolGroups';
+  }
   await checkAuthResourceMigration(context, selectedAuthResource);
-  const backendDir = pathManager.getBackendDirPath();
 
-  const destPath = path.normalize(path.join(backendDir, category, selectedAuthResource));
-  const srcPath = path.normalize(path.join(__dirname, '..', '..', '..', 'resources', 'overrides-resource'));
+  // override structure for auth resource
+  await generateOverrideforAuthResource(context, selectedAuthResource, 'auth');
```

Comment:
wouldn't this be required to be wrapped around an if as well?

Reply:
no , if no resource is selected it return and 
if authResource selected -> skeleton generated for auth resource

if userPoolGroupResource selected -> skeleton generated for auth resource and userPool Group resource

Reply:
when userPoolGroupResource is selected why do we need to generate override for both auth resource and userpool group resource and not just user pool group resource?